javascript/283-Move-Zeroes.js

```diff
@@ -1,3 +1,27 @@
+/**
+ * Linear Time
+ * Time Complexity  O(N) | Space Complexity O(N);
+ * @param {number[]} nums
+ * @return {void} Do not return anything, modify nums in-place instead.
+ */
+var moveZeroes = function(nums) {
+
+    const zeroAtTheEnd = Array(nums.length).fill(0);
+    let left = 0;
+    let right = zeroAtTheEnd.length - 1;
+
+    for (let i = 0; i < nums.length; i++) {
+        if (nums[i] === 0) {
+            right--;
```

Comment:
Add continue an remove else branch

Reply:
@aakhtar3, I updated the code, we don't need else or the continue part now.